GenderPayGap.WebUI/Views/ChangePersonalDetails/ChangePersonalDetails.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Account.ChangePersonalDetailsViewModel
+
+@{
+    ViewBag.Title = "Change your personal details - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Change your personal details
+        </h1>
+            
+        <form method="POST" action="@Url.Action("ChangePersonalDetailsPost", "ChangePersonalDetails")">
+            @(Html.AntiForgeryToken())
+            
+            @(Html.GovUkErrorSummary())
+            
+            @(Html.GovUkTextInputFor(
+                m => m.FirstName,
+                labelOptions: new LabelViewModel { Text = "First name" },
+                autocomplete: "given-name"
+            ))
+            
+            @(Html.GovUkTextInputFor(
+                m => m.LastName,
+                labelOptions: new LabelViewModel { Text = "Last name" },
+                autocomplete: "family-name"
+                ))
+            
+            @(Html.GovUkTextInputFor(
+                m => m.JobTitle,
+                labelOptions: new LabelViewModel { Text = "Job title" },
+                autocomplete: "organization-title"
+                ))
+            
+            @(Html.GovUkTextInputFor(
+                m => m.ContactPhoneNumber,
+                labelOptions: new LabelViewModel { Text = "Contact phone number" },
+                autocomplete: "tel"
+                ))

```

Comment:
I might add some width classes to these inputs (to make them look nicer and to make it clearer how much data we're expecting)
https://design-system.service.gov.uk/components/text-input/#fixed-width-inputs

I think the "20" width would work for all of them (it doesn't strictly allow 20 characters, it's actually quite a bit wider - approx 1/2 the width of the overall page

e.g.
```cshtml
@(Html.GovUkTextInputFor(
    m => m.FirstName,
    labelOptions: new LabelViewModel { Text = "First name" },
    autocomplete: "given-name",
    classes: "govuk-input--width-20"
))
```